views/house_download.erb

```diff
@@ -0,0 +1,198 @@
+<div class="page-section page-section--grey text-center">
+  <div class="container">
+    <h3>Download data for politicians in <%= @page.house.name %>, <%= @page.country.name %>.</h3>
```

Comment:
I think this might be simpler as "Download data for politicians in this legislature." because

* you've already got the house and country name immediately above this line anyway
* it might an article (e.g., "...politicians in *the* Senate..." but I'm not sure that automatically works for every house name we have (it might)

So I suggest you simplify this with no variables in the string, and if it causes upset subsequently raise an issue to fix/refine it.